src/mmw/js/src/modeling/models.js

```diff
@@ -276,10 +275,8 @@ var ScenarioModel = Backbone.Model.extend({
         this.set('inputs', new ModificationsCollection(attrs.inputs));
         this.set('modifications', new ModificationsCollection(attrs.modifications));
 
-        this.on('change:project change:name change:census', this.attemptSave, this);
-        this.get('inputs').on('add', this.attemptSave, this);
+        this.on('change:project change:name', this.attemptSave, this);
         this.get('modifications').on('add remove change', this.updateModificationHash, this);
-        this.get('modifications').on('add remove', this.attemptSave, this);
```

Comment:
I think we'll end up tweaking the timeout on the polling. Initial results of the real geotrellis endpoint is making me think we'll be in the 20sec+ range.